lib/java/com/google/android/material/chip/ChipDrawable.java

```diff
@@ -520,11 +522,37 @@ private boolean canShowCheckedIcon() {
   /** Returns the width of the chip icon plus padding, which only apply if the chip icon exists. */
   float calculateChipIconWidth() {
     if (showsChipIcon() || (showsCheckedIcon())) {
-      return iconStartPadding + chipIconSize + iconEndPadding;
+      return iconStartPadding + currentChipIconWidth() + iconEndPadding;
     }
     return 0f;
   }
 
+  /**
+   * Returns the width of the chip using, chipIconSize, checkedIcon or chipIcon.
```

Comment:
This comment is confusing: 
1) this method does not return the width of the chip, it only returns the width of a chip's leading icon. 
2) I don't understand what "using, chipIconSize, checkedIcon or chipIcon." means

Reply:
I've changed the comments with _icon/checkIcon_ to _chip's leading icon_